src/server/cmd/mount-server/cmd/cmd.go

```diff
@@ -34,6 +36,7 @@ func MountServerCmd() *cobra.Command {
 	}
 	rootCmd.Flags().StringVar(&mountDir, "mount-dir", "/pfs", "Target directory for mounts e.g /pfs")
 	rootCmd.Flags().BoolVar(&allowOther, "allow-other", true, "Mount is created with allow-other option")
+	rootCmd.Flags().StringVar(&sockDir, "sock-dir", "/tmp", "Socket directory to be used e.g /tmp")
```

Comment:
You set the default to "/tmp" then it will be as if it were always present.  It needs to default to off to avoid breaking other uses of mount-server.

I would suggest using a full path argument --socket-path={/tmp/pfs.sock} -- so you don't need to compose it and it gives full control to the client.

Reply:
ok